<?xml version="1.0"?>
<launch>
  <node pkg="ball_picker" type="mover_test" name="mover_test" output="screen">
    <remap from="goal_coords" to="/goal_coords" />
  </node>
</launch>
